admin/services/class-indexable.php

```diff
@@ -41,7 +41,7 @@ public function get_indexable( WP_REST_Request $request ) {
 					$object_type,
 					$object_id
 				),
-				404
+				400
```

Comment:
I think this instance should remain a 404, since here the object is indeed "not found". Only using an invalid type is a "400 Bad Request" I'd say.